### PR TITLE
Delete old db before launching workspace

### PR DESCRIPTION
**Pull request by chowwinston**
@W-4199304@

### What does this PR do?
Deletes old apex.db upon before launching the language server
### What issues does this PR fix or reference?
Previously the DB used the full file path which caused some conflicts. This will ensure we start fresh with each launch

### CODE REVIEW

File: packages/salesforcedx-vscode-apex/src/languageServer.ts [hunk 1]
```diff
@@ -104,6 +106,14 @@ async function createServer(
   }
 }
 
+function deleteDbIfExists(): void {
+  const workspace = vscode.workspace.rootPath;
+  const dbPath = workspace + '/.sfdx/tools/apex.db';
+  if (workspace !== undefined && fs.exists(dbPath)) {
+    fs.unlinkSync(dbPath);
```
Comment (vazexqi) [inline packages/salesforcedx-vscode-apex/src/languageServer.ts]: Can you check that this works on Windows. The process has not started yet so this should work but it's best to check.

File: packages/salesforcedx-vscode-apex/src/languageServer.ts [hunk 2]
```diff
@@ -104,6 +106,14 @@ async function createServer(
   }
 }
 
+function deleteDbIfExists(): void {
+  const workspace = vscode.workspace.rootPath;
+  const dbPath = workspace + '/.sfdx/tools/apex.db';
```
Comment (vazexqi) [inline packages/salesforcedx-vscode-apex/src/languageServer.ts]: Does this require changes on the Apex jar or are you just deleting this first to be safe until we fix that?

Comment (chowwinston) [inline packages/salesforcedx-vscode-apex/src/languageServer.ts]: Just deleting to be safe. It should be fixed when the new jar goes out.

File: packages/salesforcedx-vscode-apex/src/languageServer.ts (hunk 1 above)
Comment (vazexqi) [inline packages/salesforcedx-vscode-apex/src/languageServer.ts]: Also, don't you need to use `fs.existsSync` instead of `fs.exists`?

Comment (vazexqi) [inline packages/salesforcedx-vscode-apex/src/languageServer.ts]:
```
  if (vscode.workspace.rootPath) {
    const dbPath = path.join(
      vscode.workspace.rootPath,
      '.sfdx',
      'tools',
      'apex.db'
    );
    if (fs.existsSync(dbPath)) {
      fs.unlinkSync(dbPath);
    }
  }
```

**Pull request merged**